apps/bundle_deploy/bundle.c

```diff
@@ -28,6 +29,11 @@
 #include "backtrace.h"
 #endif
 
+#define CRT_MEMORY_NUM_PAGES 16384
+#define CRT_MEMORY_PAGE_SIZE_LOG2 10
```

Comment:
consider making this as part of `crt_config.h` ?

Reply:
so actually i'm going the other way with this PR--saying that the backing memory for the memory allocator needs to be allocated by the caller. we can put these into `crt_config.h`, but I'm not sure what the benefit is after this change--the total memory is now passed in to memory.c as an argument.